README.md

```diff
@@ -4,13 +4,13 @@ Gazebo - A dynamic multi-robot simulator
 This is the Gazebo simulator.  Gazebo simulates multiple robots in a
 3D environment, with extensive dynamic interaction between objects.
 
-  http://gazebosim.org
+  http://classic.gazebosim.org
 
 Installation
 ------------
 Instructions are located at
 
-  http://gazebosim.org/install
+  http://classic.gazebosim.org/install
```

Comment:
I think this link is broken? Could be an issue with the website

Reply:
I'm curious how the uninstall link goes to 'uninstall from source' but not the install one

Reply:
here's the redirect: https://github.com/osrf/gazebo_web_app/blob/master/config/routes.rb#L48

Reply:
we probably just need to update the redirect link there

Reply:
I can take that up